callstats-java-sdk/src/main/resources/log4j2.xml

```diff
@@ -9,9 +9,13 @@
         </File>
     </Appenders>
     <Loggers>
-    <Logger name="org.apache.log4j.xml" level="debug"/>
+    	<Logger name="org.apache.log4j.xml" level="debug"/>
+    	<Logger name="org.apache.log4j.xml" level="info"/>
         <Root level="debug">
             <AppenderRef ref="trace-log"/>
         </Root>
+        <Root level="info">
```

Comment:
This log4j configuration seems a bit confusing. Seems to me it's almost like the second line immediately override the previous line. Can you double check it?